### PR TITLE
💃  Add more collaboration options

### PR DESCRIPTION
**Pull request by jasonLaster**
Associated Issue: #1872 

 :1st_place_medal: 

CC @clarkbw 

@bomsy, @wldcordeiro, @Zacqary, @jbhoosreddy, @arthur801031, @clarkbw are you okay fielding questions on slack?

### CODE REVIEW

File: docs/local-development.md [hunk 1]
```diff
@@ -549,9 +550,52 @@ When we started the Debugger, we were not sure if we would keep webpack and the
 
 Now that we *have* decided that working in github with webpack has a lot of benefits we could switch to JSX. We are open to switching if someone could help us do it, join the [discussion here](https://github.com/devtools-html/debugger.html/issues/1747).
 
+
+### Ninjas
```
Comment (jasonLaster) [inline docs/local-development.md]: I went with ninja because I thought there would be a cute image we could use. there's probably a better name tho...

Comment (jasonLaster) [inline docs/local-development.md]: What do people think of the idea generally?

File: README.md [hunk 2]
```diff
@@ -32,26 +32,30 @@ cd debugger.html
...
-* practice [debugging the debugger][first-activity]
-* claim an [up for grabs][up-for-grabs] issues
-* read the [app overview][app-overview] or [contributing][contributing] guidelines
-* watch a [screencast][getting-started-screencast]
-* have a [look at the features and products page][tracking] to track what we're working on
+* Claim an [up for grabs][up-for-grabs] issues. If you get stuck, ask a [ninja][ninja].
+* Do our getting started activity [debugging the debugger][first-activity]
+* Read the [app overview][app-overview] or [contributing][contributing] guidelines
+* Watch a [video][getting-started-screencast] on contributing to the Debugger.
+* Go to the [features[tracking] board to see what we're working on
```
Comment (clarkbw) [inline README.md]: Small typo, missing a `]` to close it.

File: README.md [hunk 3]
```diff
@@ -70,7 +74,7 @@ Say hello in [Slack][slack] or in the [#devtools-html][irc-devtools-html] channe
 [app-overview]:./docs/debugger-html-react-redux-overview.md
 [first-activity]:./docs/debugging-the-debugger.md
 [tracking]:https://github.com/devtools-html/debugger.html/projects/10
-
+[ninja][ninja]:./docs/local-development.md#ninja
```
Comment (clarkbw) [inline README.md]: `[ninja]`s, there can be only one!  🤕

File: docs/local-development.md (hunk 1 above)
Comment (clarkbw) [inline docs/local-development.md]: In general good, not sure about the Ninja part.  I like that its not `maintainers` which always sounds pretty lame.  Keep it fun!  Maybe lets keep asking around for more ideas.

Comment (jasonLaster) [inline docs/local-development.md]: yep - i think we can do better than ninja too :)

Comment (Garbee) [inline docs/local-development.md]: Guru. We could get like, a tranquil monk picture.

No partisanship here... :laughing:

Comment (wldcordeiro) [inline docs/local-development.md]: I like this idea, and I'm happy to help other contributors. 👍

**Comment by jasonLaster** [comment]
@gregor any suggestions on ninja naming? 

The issue describes the goal. We want to help people feel comfortable asking for help

File: README.md (hunk 3 above)
Comment (clarkbw) [inline README.md]: But seriously I think you mean: `[ninja]:./docs/local-development.md#ninja` 🎃

**Comment by jasonLaster** [comment]
Hmm, so wizards could be a good term. But I think it might promote a status symbol that I'm not as comfortable with. 

Perhaps we describe ppl as "helpful folks". I think the TOC could be "help". And in the body of the section say we have some helpful folks...

File: docs/local-development.md (hunk 1 above)
Comment (arthur801031) [inline docs/local-development.md]: I'm happy to help people on Slack! 👍👍😃

**Comment by gr2m** [comment]
@jasonLaster in general we try very hard to avoid anything that could lead towards a hero culture. "ninja" or "rockstar" have a bad sound due to their over-usage in hiring advertisement which are nearly all tailored towards white mid-aged male developers. 

A term I have seen that might work better is "champion". If I recall correctly the Ember community is using the term for their RFCs: each request for change needs a "champion" who is in charge of managing the process. I agree that the word by itself could also lead to a hero culture, but it’s on us to give it a positive meaning by using it in a way that _always puts the community first_, not the person who is "championing" a project. Champions should serve the community and let over people shine, not the over way around.

I hope this is helpful, please let me know if something is not clear. Note that English is not my native language, it’s a bit hard for me to express myself

**Comment by jasonLaster** [comment]
@gr2m that helps a lot. I *do* want to avoid hero culture. I like the context for *champion* in the case of championing a cause like pushing an RFC concept here.

 I'm happy with putting the action first "getting help" and then using friendly language *folks* and *emojis*. 

Perhaps we could reach for some educator terms if we wanted too, but I'm not sure if we need it. I'm thinking of tutor...

Here's what we have now. I'm pretty happy with it

screen shot 2017-01-28 at 7 28 20 pm

**Comment by gr2m** [comment]
The emojis are great :) Thanks for addressing my concerns <3

**Pull request merged**